app/controllers/user_controller.py

```diff
@@ -0,0 +1,18 @@
+from typing import Tuple
+from flask import Blueprint
+from app.models import User
+from app.utils.session_decorators import session_required
+
+
+user_routes = Blueprint('user_routes', __name__)
+
+
+@user_routes.route('/<username>')
```

Comment:
Can you define the http verbs this accepts, so just GET for this one like
```suggestion
@user_routes.route('/<username>', methods=['GET'])
```